samples/mysql-schema/src/main/resources/application.properties

```diff
@@ -4,4 +4,5 @@ quarkus.native.additional-build-args=--initialize-at-run-time=org.apache.commons
 quarkus.datasource.username=root
 quarkus.datasource.devservices.image-name=mariadb:10.7
 quarkus.kubernetes-client.devservices.override-kubeconfig=true
-quarkus.http.test-port=0
\ No newline at end of file
+quarkus.http.test-port=0
+quarkus.kubernetes.image-pull-policy=IfNotPresent
```

Comment:
Why was this added?

Reply:
I think that was just to test with minikube locally, can remove for now

Reply:
done